prusa_slicer_post_processing_script.py

```diff
@@ -1189,9 +1190,21 @@ def _warning(message,category = UserWarning, filename = '', lineno = -1,*args, *
 
 ################################# MAIN EXECUTION #################################
 ##################################################################################
-if __name__=="__main__":
-    gCodeFileStream,path2GCode = getFileStreamAndPath()
-    skipInput=False
-    if platform.system()!="Windows":
-        skipInput=True
-    main(gCodeFileStream,path2GCode, skipInput)
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Process G-code files.")
```

Comment:
This description is vague and maybe a little lazy. Maybe something like "Process overhangs within PrusaSlicer G-code files into circular arcs." would be clearer.